task1.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <link href="task1.css" rel="stylesheet">
+    <meta charset="utf-8">
+    <title>Flowers</title>
+</head>
+<body>
+  <p>
+        <div>
+          <h1>Flowers</h1>
+        </div>
+  <div class="date">
+      <p>November 24th,2018</p>
+  </div>
+  <div class="first">
+      <p><img src="email.png.png" alt="email.png.png" width="40">

```

Comment:
filename = any name + extension name
Best practice:
```suggestion
      <p><img src="email.png" alt="email" width="40">
```